obs-studio-server/source/util-crashmanager.cpp

```diff
@@ -238,10 +241,63 @@ nlohmann::json RequestProcessList()
 //////////////////
 // CrashManager //
 //////////////////
+#ifdef WIN32
+bool util::CrashManager::IsMemoryDumpEnabled()
+{
+	if (std::filesystem::exists( memoryDumpFolder ) && std::filesystem::is_directory( memoryDumpFolder )) {
+		return true;
+	} else {
+		return false;
+	}
+}
 
-bool util::CrashManager::Initialize(char* path, std::string appdata)
+bool util::CrashManager::InitializeMemoryDump()
+{
+	bool ret = false;
+	if (!IsMemoryDumpEnabled())
+		return ret;
+
+	PSECURITY_DESCRIPTOR securityDescriptor = (PSECURITY_DESCRIPTOR) LocalAlloc(LPTR, SECURITY_DESCRIPTOR_MIN_LENGTH);
+	if (InitializeSecurityDescriptor(securityDescriptor, SECURITY_DESCRIPTOR_REVISION)) {
+		if (SetSecurityDescriptorDacl( securityDescriptor, TRUE, NULL, FALSE)) {
+			SECURITY_ATTRIBUTES eventSecurityAttr = {0};
+			eventSecurityAttr.nLength = sizeof(SECURITY_ATTRIBUTES);
+			eventSecurityAttr.lpSecurityDescriptor = securityDescriptor;
+			eventSecurityAttr.bInheritHandle = FALSE;
+
+			memoryDumpEvent = CreateEvent( &eventSecurityAttr, TRUE, FALSE, L"Global\\OBSMEMORYDUMPEVENT");
```

Comment:
Might be better to call it `SLOBSMEMORYDUMPEVENT` so there is no confusion.

Reply:
done.